blocks/core/ff_module/ff_module-response-recipient-list/ff_module-response-recipient-list.js

```diff
@@ -1,13 +1,34 @@
 'use strict';
 
 var React = require('react');
-var template = require('./_ff_module-response-recipient-list.rt.js');
+var ProfileResponseButton = require('../ff_module-profile-response-button/ff_module-profile-response-button.js');
+var ItemRepeater = require('../../ff_container/ff_container-item-repeater/ff_container-item-repeater.js');
 
 module.exports = React.createClass({
     displayName: 'TaskResponseRecipientList',
     propTypes:{
         responses: React.PropTypes.array.isRequired,
         onSelect: React.PropTypes.func.isRequired
     },
-    render: template
+    render: function() {
+    	return <ItemRepeater modifier='separated'>
+    		{this.props.responses.map((response)=>{
+    			var currentResponse = response.currentTaskResponse ? <div className='ff_module-response-recipient-list__currentResponse'>{response.currentTaskResponse}</div> : '';
```

Comment:
The currentResponse ought be a prop on the main component, not the responses - it's just a single item, like the 'details'